src/BookYourShow.Api/Controllers/GenresController.cs

```diff
@@ -0,0 +1,76 @@
+﻿using BookYourShow.Models;
+using BookYourShow.Api.Repository;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BookYourShow.Api.Controllers
+{
+    [Route("api/[controller]")]
```

Comment:
follow api naming conventions, `api/[controller]` to `genere`

Reply:
api names changed